docs/modules/ROOT/pages/how-tos/cloudscale/setup-storage-cluster.adoc

```diff
@@ -0,0 +1,363 @@
+  # (config refers to values not yet known)
+ <= data "local_file" "hieradata_mr_url"  {
+      + content        = (known after apply)
+      + content_base64 = (known after apply)
+      + filename       = "/builds/syn/cluster-catalogs/c-cluster-id-1234/manifests/openshift4-terraform/.mr_url.txt"
+      + id             = (known after apply)
+    }
+  # module.cluster.module.lb.module.hiera[0].gitfile_checkout.appuio_hieradata will be created
+  + resource "gitfile_checkout" "appuio_hieradata" {
+      + branch = "master"
+      + head   = (known after apply)
+      + id     = (known after apply)
+      + path   = "./appuio_hieradata"
+      + repo   = "https://project_368_bot@git.vshn.net/appuio/appuio_hieradata.git"
+    }
+  # module.cluster.module.lb.module.hiera[0].local_file.lb_hieradata will be created
+  + resource "local_file" "lb_hieradata" {
+      + content              = <<-EOT
+            # Managed by Terraform for Project Syn cluster c-cluster-id-1234
+            profile_openshift4_gateway::nodes:
+              - lb-XX.cluster-id-1234.example.com
+              - lb-YY.cluster-id-1234.example.com
+            profile_openshift4_gateway::public_interface: ens3
+            profile_openshift4_gateway::private_interfaces:
+              - ens4
+            profile_openshift4_gateway::floating_addresses:
+              api: <API VIP>
+              nat: <NAT VIP>
+              router: <ROUTER VIP>
+            profile_openshift4_gateway::floating_address_provider: cloudscale
+            profile_openshift4_gateway::internal_vip: 172.18.200.100
+            profile_openshift4_gateway::floating_address_settings:
+              token: [MASKED]
+            profile_openshift4_gateway::backends:
+              'api':
+                - etcd-0.c-cluster-id-1234.example.com
+                - etcd-1.c-cluster-id-1234.example.com
+                - etcd-2.c-cluster-id-1234.example.com
+              'router':
+                - 172.18.200.175
+                - 172.18.200.112
+                - 172.18.200.160
+        EOT
+      + directory_permission = "0755"
+      + file_permission      = "0644"
+      + filename             = "/builds/syn/cluster-catalogs/c-cluster-id-1234/manifests/openshift4-terraform/appuio_hieradata/lbaas/c-cluster-id-1234.yaml"
+      + id                   = (known after apply)
+    }
+Plan: 9 to add, 0 to change, 0 to destroy.
+----
+
+. Approve node certificates for the storage nodes
++
+include::partial$install/approve-node-csrs.adoc[]
+
+. Label and taint the storage nodes
++
+include::partial$label-taint-storage-nodes.adoc[]
+
+. Configure the storage cluster
++
+[TIP]
+====
+The bulk of the configuration for the `rook-ceph` component is done in the https://git.vshn.net/syn/commodore-defaults/-/blob/490282a1b70bd3a79bf95b222c2bec44cda60bb6/cloud/cloudscale/params.yml#L58-96[VSHN global defaults repo (internal)].
```

Comment:
I'm unsure what I think of that link. On the one side we need to have a commit pinned link so that the line numbers are correct. But when I click on that link, I want to see the current config not the one added initially. Maybe it would make more sense to just link the file at the current `master`?

Reply:
Yeah, I wasn't too happy with it either, I'll change it to just point to the file in the master branch.